students/martin-yela/mi-primer-html.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1 id="inicio">Mi primer HTML</h1>
+    <h2>Esto es el primer párrafo</h2>
+        <p>Lorem ipsum dolor sit <b>consectetur adipisicing</b> elit. Illo amet <i> cum quae reprehenderit!</i> 
+            Odit eveniet autem cum unde saepe commodi <u> rem veritatis perferendis</u> nam itaque repellat nesciunt provident, 
+            in qui!</p>
+    <h2>Mi primera lista</h2>
+            <ul>
+                <li><a href="https://www.google.es">Ir a google</a></li>
+                <li><a href="mailto:martinyela@hotmail.com">Enviar un mail</a></li>
+                <li><a href="tel:+34628668246">Llamar por teléfono</a></li>
```

Comment:
Be careful with your personal information in public repositories. If it is your phone number, change it to a fake number